### ISSUE TITLE
0.4.0 no worked

### ISSUE THREAD

**Issue by SergeyDjam** [FreeTubeApp/FreeTube#186]
I have tried AppImage, have tried to collect a package and AppImage in the OS, tried to start from the folder from the program the npm start team, but nothing has helped.

> ./FreeTube-0.4.0-x86_64.AppImage 
/tmp/.mount_FreeTu2SeTlM/FreeTube: /lib64/libdbus-1.so.3: no version information available (required by /tmp/.mount_FreeTu2SeTlM/FreeTube)
/tmp/.mount_FreeTu2SeTlM/FreeTube: /lib64/libdbus-1.so.3: no version information available (required by /tmp/.mount_FreeTu2SeTlM/FreeTube)
(electron) The 'lstatSyncNoException' function has been deprecated and marked for removal.
(electron) 'app.makeSingleInstance(cb)' is deprecated. Use 'app.requestSingleInstanceLock() and app.on('second-instance', cb)' instead.

But 0.3.2 no poblem

**Comment by SergeyDjam** [comment]
Release 0.4.1 Beta - fix

**Comment by SergeyDjam** [comment]
no fix in 0.4.1 beta

```
./FreeTube
./FreeTube: /lib64/libdbus-1.so.3: no version information available (required by ./FreeTube)
/mnt/cl/home/admin/rpmbuild/BUILD/FreeTube-0.4.1-beta/out/FreeTube-linux-x64/FreeTube: /lib64/libdbus-1.so.3: no version information available (required by /mnt/cl/home/admin/rpmbuild/BUILD/FreeTube-0.4.1-beta/out/FreeTube-linux-x64/FreeTube)
(electron) The 'lstatSyncNoException' function has been deprecated and marked for removal.
(electron) 'app.makeSingleInstance(cb)' is deprecated. Use 'app.requestSingleInstanceLock() and app.on('second-instance', cb)' instead.
```
```
npm start

> FreeTube@0.4.1 start /mnt/cl/home/admin/rpmbuild/BUILD/FreeTube-0.4.1-beta
> electron-forge start

✔ Checking your system
✔ Locating Application
✔ Preparing native dependencies
✔ Launching Application
/mnt/cl/home/admin/rpmbuild/BUILD/FreeTube-0.4.1-beta/node_modules/electron-prebuilt-compile/node_modules/electron/dist/electron: /lib64/libdbus-1.so.3: no version information available (required by /mnt/cl/home/admin/rpmbuild/BUILD/FreeTube-0.4.1-beta/node_modules/electron-prebuilt-compile/node_modules/electron/dist/electron)
/mnt/cl/home/admin/rpmbuild/BUILD/FreeTube-0.4.1-beta/node_modules/electron-prebuilt-compile/node_modules/electron/dist/electron: /lib64/libdbus-1.so.3: no version information available (required by /mnt/cl/home/admin/rpmbuild/BUILD/FreeTube-0.4.1-beta/node_modules/electron-prebuilt-compile/node_modules/electron/dist/electron)
(electron) The 'lstatSyncNoException' function has been deprecated and marked for removal.
(electron) The 'lstatSyncNoException' function has been deprecated and marked for removal.
(electron) The 'lstatSyncNoException' function has been deprecated and marked for removal.
(electron) 'app.makeSingleInstance(cb)' is deprecated. Use 'app.requestSingleInstanceLock() and app.on('second-instance', cb)' instead.
```

**Comment by SergeyDjam** [comment]
Worked only on electorn 2.0.6.
